Casks/d/digiexam.rb

```diff
@@ -0,0 +1,31 @@
+cask "digiexam" do
+  version "25.3.67"
+  sha256 :no_check
+
+  url "https://www.digiexam.com/hubfs/client/Digiexam_Mac_universal.dmg"
+  name "digiexam"
+  desc "Academic testing platform with device lockdown"
+  homepage "https://www.digiexam.com/"
+
+  livecheck do
+    url :url
+    strategy :extract_plist
+  end
```

Comment:
I left the comments in here, we can remove them if needed, but would appreciate our resident livecheck guru @samford to take a look.  I modeled this on some other livechecks, but worry it could be fragile.

```suggestion
  livecheck do
    url "https://www.digiexam.com/release-notes"
    strategy :page_match do |page|
      # Digiexam loads release notes from a dynamic JavaScript file
      js_match = page.match(%r{src=["']?([^"' >]*?module_Release_Notes_Customized_Module\.min\.js)["']?}i)
      next if js_match.blank?

      js_url = js_match[1]
      js_content = Homebrew::Livecheck::Strategy.page_content(js_url)
      next if js_content.blank?

      # Find the Mac section and extract the latest version
      # Format: mac:[{quarter:"...",releases:[{version:"25.3.67",...
      js_content[:content]&.scan(/mac:\[.*?version:"(\d+(?:\.\d+)+)"/i)&.map { |match| match[0] }
    end
  end
```

Reply:
There's also a [help article](https://support.digiexam.se/hc/en-us/articles/5714871263388-Supported-and-deprecated-client-and-OS-versions) that lists the latest version but that may become outdated over time, so I think checking the release notes page makes sense.

There are some changes that I would like to make to this suggestion but it's quite late over here, so I'll come back to this in the morning.

Reply:
I pushed a commit that contains a modified version of this approach. I agree with the general idea, so I mostly tweaked it a bit. Notable changes:

* Made the JS-matching regex a little more flexible (it could still easily break if the file name format notably changes).
* Used `[..., 1]` instead of `#match`/`js_match[1]`, for the sake of brevity.
* Used `URI.join(@url, js_url).to_s` to ensure this would continue to work if/when the JS file URL becomes relative rather than absolute (as it is now).
* Moved the version-matching regex out of the `strategy` block into a `regex()` call, as it's the main regex that's used for matching version information.
* Tweaked the version-matching regex to be a little more flexible (i.e., accommodating whitespace, matching both `"` and `'`, omitting a leading `v` if they ever switch to that format, etc.).